src/Makevars.win

```diff
@@ -0,0 +1,12 @@
+PKG_CPPFLAGS = -DARMA_64BIT_WORD=1 -I.
+PKG_CXXFLAGS = -std=gnu++11  -g -O2 -I.
+CXX = /usr/local/opt/llvm/bin/clang++
+CXX11 = clang++
+PKG_LIBS = -lz   -lz -L/usr/local/Cellar/r/4.2.1_4/lib/R/lib $(LAPACK_LIBS) -L/usr/local/opt/gcc/lib/gcc/current/gcc -L/usr/local/opt/gcc/lib/gcc/current -lgfortran -lquadmath $(BLAS_LIBS)
```

Comment:
Did the openMP part work before?

Reply:
Are you referring to the first version of the Makevars.win file I had? That came from Dirk Eddelbuettel's reponse on this thread https://stackoverflow.com/questions/32974568/r-package-with-c-c-and-openmp-how-to-make-makevars-file-under-mypackage-sr . I think it looked something like:

```
## CXX_STD = CXX11
## PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)
## PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)
```

That worked but then it was missing the methods you redefined if I understood correctly. I used what that first version was using to find BLAS in order to have the Makevars.win I made from your Makevars find BLAS.

Reply:
☝️ if that is what you meant, I updated the Makevars.win file to use that notation instead (it seems more compact) and it still passes the tests. Please double check it before merging to see if something looks particularly off or missing (you can check the diff in the very last commit to compare both).